extensions/binding/org.eclipse.smarthome.binding.weatherunderground/src/main/java/org/eclipse/smarthome/binding/weatherunderground/internal/WeatherUndergroundHandlerFactory.java

```diff
@@ -28,17 +34,20 @@
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.Reference;
 
+import com.google.common.collect.Sets;
```

Comment:
we try not to use google guava for new contributions. See the next comment for an alternative.